mempool/v1/mempool_test.go

```diff
@@ -306,11 +306,11 @@ func TestTxMempool_Flush(t *testing.T) {
 }
 
 func TestTxMempool_ReapMaxBytesMaxGas(t *testing.T) {
+	totalSizeBytes := int64(5800)
```

Comment:
[Question] Where does the 5800 value come from? would be good to add a comment in the code

Reply:
yeah true, 

While its something we might want to know in the future, the exact overhead introduced by the mempool and encoding isn't specified already and would take a while to calculate accurately.

The txs being used are 38 bytes (and thanks to this fix they are actually consistent), and there should be 100 of them. Presumably the rest is encoding overhead.

Reply:
added docs here tho https://github.com/celestiaorg/celestia-core/pull/1170/commits/dbbd7eeed803cc9d2af1d11524b76fafb29c1f32